certbot-nginx/certbot_nginx/nginxparser.py

```diff
@@ -4,7 +4,7 @@
 import string
 
 from pyparsing import (
-    Literal, White, Word, alphanums, CharsNotIn, Forward, Group,
+    Literal, White, Word, alphanums, CharsNotIn, Combine, Forward, Group,
```

Comment:
@pde. Is this funny (like should we laugh emoji?) because the relationship between these pyparsing 'words' is non-obvious